test/controllers/api/v1/stats/channels_controller_test.rb

```diff
@@ -0,0 +1,35 @@
+require "test_helper"
+require "shared/mailer_test_helper"
+
+class Api::V1::Stats::ChannelsControllerTest < ActionDispatch::IntegrationTest
+  test "/api/v1/stats/channels returns list of all channel uuids" do
+
+    get "/api/v1/stats/channels", headers: { "HTTP_AUTHORIZATION" => "Token token=fake_api_auth_token" }
+    result = JSON.parse(response.body)
+
+    assert result.is_a?(Array)
+    assert_equal(Channel.count, result.length)
+
+  end
+
+  test "/api/v1/stats/channels/:uuid returns json representation of channel, or 404 if not found" do
+
+    if(Channel.last != nil)
```

Comment:
You should be creating a `Channel` to test against

Reply:
Added ✔️